plugins/woocommerce-admin/client/homescreen/mobile-app-modal/layouts/ModalIllustrationLayout.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import type React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Illustration from '../illustrations/intro-devices-desktop.png';
+
+export const ModalIllustrationLayout = ( {
+	body,
+}: {
+	body: React.ReactNode;
+} ) => {
+	return (
+		<div className="mobile-app-modal-layout">
+			<div className="mobile-app-modal-content">{ body }</div>
+			<div className="mobile-app-modal-illustration">
+				<img
+					src={ Illustration }
+					alt="Screen captures of the WooCommerce mobile app"
```

Comment:
It may be worth putting this in `__()`.

```suggestion
					alt={ __( 'Screen captures of the WooCommerce mobile app', 'woocommerce' ) }
```